webpack/global_test_setup.js

```diff
@@ -0,0 +1,6 @@
+// runs before each test to make sure console.error output will
+// fail a test (i.e. default PropType missing). Check the error
+// output and traceback for actual error.
+global.console.error = (error) => {
```

Comment:
I don't sure we actually need this file, in core it just works fine

Reply:
thanks, removed it and instead using only `test_setup.js`